e2e/stories/create-record.e2e.ts

```diff
@@ -43,6 +43,21 @@ describe("create record REST API", async () => {
     );
   });
 
+  it("create a record with single object notation return array with proper record", async () => {
```

Comment:
... notation *should* return an array ...

Reply:
done